From: Thomas Hindoe Paaboel Andersen <phomes@gmail.com>
Date: Sat, 31 May 2014 21:36:23 +0200
Subject: util: ignore_file should not allow files ending with '~'

ignore_file currently allows any file ending with '~' while it
seems that the opposite was intended:
a228a22fda4faa9ecb7c5a5e499980c8ae5d2a08

(cherry picked from commit 93f1a06374e335e8508d89e1bdaadf45be6ab777)
(cherry picked from commit c4a42680d72b18c5bd8717e6cff3e0df049936d9)
---
 src/shared/util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/shared/util.c b/src/shared/util.c
index aa6242d..6e97a6d 100644
--- a/src/shared/util.c
+++ b/src/shared/util.c
@@ -1499,7 +1499,7 @@ bool ignore_file(const char *filename) {
         assert(filename);
 
         if (endswith(filename, "~"))
-                return false;
+                return true;
 
         return ignore_file_allow_backup(filename);
 }
